messaging-clustering-singleton/README-source.adoc

```diff
@@ -468,20 +500,78 @@ This script removes the server configuration that was done by the install-standa
 The batch executed successfully
 ----
 
-=== Delete the Cloned Standalone {productName} Directory
+==== Delete the Cloned Standalone {productName} Directory
 
 . If it is running, stop the second instance of the {productName} server.
 . Delete the cloned directory.
 
-// Run the Quickstart in Red Hat CodeReady Studio or Eclipse
-include::../shared-doc/run-the-quickstart-in-jboss-developer-studio.adoc[leveloffset=+1]
+// Build and run sections for other environments/builds
+ifndef::ProductRelease,EAPXPRelease[]
 
-//*************************************************
-// Product Release content only
-//*************************************************
-ifdef::ProductRelease[]
+[[build_and_run_the_quickstart_with_provisioned_server]]
+== Building and running the quickstart application with provisioned {productName} server
+
+Instead of using a standard {productName} server distribution, the two {productName} servers to deploy and run the quickstart can be alternatively provisioned by activating the Maven profile named `provisioned-server` when building the quickstart:
+[source,sh,subs="+quotes,attributes+",options="nowrap"]
+----
+$ mvn clean package -Pprovisioned-server
+$ mvn package -Pprovisioned-server \
```

Comment:
I think you could instead configure 2 plugin executions in the provisioned-server profile, so you would provision both using single command 'mvn clean package -Pprovisioned-server'.

For the first execution you would configure <provisioningDir>server1</provisioningDir>, and <provisioningDir>server2</provisioningDir> for the second.

PS: please note "server" to "server1" move that would affect all instructions after too

Reply:
Done. Just a note that I would keep it as `server` and `server2` to be similar with other QSs. WDYT?

Reply:
fine with me